### ISSUE TITLE
Not working for Mac OS X 10.14 Mojave

### ISSUE THREAD

**Issue by sentient06** [marczak/johnnynash#4]
It yields a black screen.

**Comment by marczak** [comment]
Yeah, I’ve looked at this a little bit, and want to check with some people at Apple, but I don’t think this is feasible any longer. The security protections are too tight now.
